### PR TITLE
APS-2727, Added new release notes for NS to GW 

### PR DESCRIPTION
**Pull request by SateeshFW**
Added release notes for Sept release and also updated the index page with the new release page

### CODE REVIEW

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  * For existing Namespaces, the default display name will be `<IDIR>'s Gateway`.
+* The Namespace `name` attribute has been renamed to `gatewayId` for Gateways.
+  * This change aims to prevent confusion between `gatewayId` and the new `displayName` attribute for Gateways.
+  * The gatewayId is still used for setting the current Gateway in the CLI (`gwa config set gateway <gatewayId>`) and in tags within Gateway Service configuration (`tags: [ ns.<gatewayId> ]`).
+* Gateway selection user interface has changed.
+  * A new **My Gateways** page has been added, featuring a searchable list of Gateways.
+  * The Gateway dropdown selector has been relocated and updated. It now appears only on relevant API provider pages.
+* New starting page for new users after API provider login.
+  * Users without any Gateways are directed to https://api.gov.bc.ca/manager/gateways/get-started.
+  * This page provides a basic introduction and instructions for the new Quick Start process for API providers.
```
Comment (timng-bcgov) [inline documentation/reference/releases/2024-sept-3.0.0.md:28]:
```suggestion
  * This page provides a basic introduction and instructions for the new Quick Start tutorial for API providers.
```

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  * The new Quick Start tutorial offers a faster and simpler option to begin with before progressing to the [Protect an API with Client Credential Flow](https://developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/tutorials/protect-client-cred/) tutorial (previously called Quick Start). It utilizes a new `quick-start` template.
```
Comment (timng-bcgov) [inline documentation/reference/releases/2024-sept-3.0.0.md:29]:
```suggestion
  * The new Quick Start tutorial offers a simpler option to begin with before progressing to the [Protect an API with Client Credential Flow](https://developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/tutorials/protect-client-cred/) tutorial (previously called Quick Start). 
  * The revised Quick Start tutorial utilizes a new `quick-start` configuration template.
```

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  * Added breadcrumbs on Gateway management pages.
+  * Gateways is no longer visible in the navigation bar for non-IDIR users (BCeID, BC Services Card, GitHub).
+  * The **Namespace Access** page is renamed to **Administration Access**.
+
+## GWA-CLI V3 
+Gateway Command Line Interface Version 3 (GWA-CLI V3.0.1) is a new and enhanced tool for configuring your API Gateway settings. Key improvements include:
+
+* Capability to create a gateway directly from the command line. 
+* New default behaviour (`gwa gateway create`) is to autogenerate a unique `gatewayId` and prompt for a custom display name.
+* `--gateway-id` and `--display-name` can be used to specify these attributes.
```
Comment (timng-bcgov) [inline documentation/reference/releases/2024-sept-3.0.0.md:40]:
```suggestion
  * The `--gateway-id` and `--display-name` flags can be used to specify these attributes.
```

**Review by timng-bcgov — COMMENTED** [review]
I suggest using [Markdownlint ](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) to lint & wrap any lines that are longer than 80 characters.

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+title: 2024-SEPT Release 3.0.0
+order: 20240919
+date: 2024-SEPT-19
+---
+
+# API Services Portal Release 3.0.0
+
+The API Services Portal has been updated with enhancements in the existing features.
```
Comment (rustyjux) [inline documentation/reference/releases/2024-sept-3.0.0.md:9]: Adding a little high level summary.
```suggestion
The API Services Portal has been updated with enhancements to improve the API provider experience. 
The core of these updates is the renaming of 'Namespaces' to 'Gateways' throughout the the API Services Portal.
```

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+* Additional Changes:
+  * Added breadcrumbs on Gateway management pages.
+  * **Gateways** is no longer visible in the navigation bar for non-IDIR users (BCeID, BC Services Card, GitHub).
+  * The **Namespace Access** page is renamed to **Administration Access**.
+
+## GWA-CLI V3
+
+Gateway Command Line Interface Version 3 (GWA-CLI V3.0.1) is a new and enhanced tool for configuring your API Gateway settings. Key improvements include:
+
+* Capability to create a gateway directly from the command line.
```
Comment (rustyjux) [inline documentation/reference/releases/2024-sept-3.0.0.md:53]:
```suggestion
```
This is not new.

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+* New default behaviour (`gwa gateway create`) is to autogenerate a unique `gatewayId` and prompt for a custom display name.
+* The `--gateway-id` and `--display-name` flags can be used to specify these attributes.
```
Comment (rustyjux) [inline documentation/reference/releases/2024-sept-3.0.0.md:56]:
```suggestion
```
Too much detail for here.

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+* Improvements to error reporting for Gateway Service publication (`gwa apply` and `gwa publish-gateway`). Additional error messaging is shown by default to support troubleshooting.
```
Comment (rustyjux) [inline documentation/reference/releases/2024-sept-3.0.0.md]:
```suggestion
* Improvements to error reporting for Gateway Service publication (`gwa apply` and `gwa publish-gateway`) to support troubleshooting.
```

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+* Improvements to Gateway Service configuration creation using templates (`gwa generate-config`). Gateway `service` and `route` names are validated for availability during configuration creation.
```
Comment (rustyjux) [inline documentation/reference/releases/2024-sept-3.0.0.md]:
```suggestion
* Improvements to Gateway Service configuration creation using templates (`gwa generate-config`). 
  * Gateway `service` and `route` names are validated for availability during configuration creation.
```

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+* Templates for implementing common authorization patterns on the API Gateway.
```
Comment (rustyjux) [inline documentation/reference/releases/2024-sept-3.0.0.md:63]:
```suggestion
```
This is not new. There is a new template `quick-start` but it's captured above in Portal.

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+  * The new Quick Start tutorial offers a simpler option to begin with before progressing to the [Protect an API with Client Credential Flow](https://developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/tutorials/protect-client-cred/) tutorial (previously called Quick Start).
+  * The revised Quick Start tutorial utilizes a new `quick-start` configuration template.
+* Additional Changes:
+  * Added breadcrumbs on Gateway management pages.
+  * **Gateways** is no longer visible in the navigation bar for non-IDIR users (BCeID, BC Services Card, GitHub).
+  * The **Namespace Access** page is renamed to **Administration Access**.
+
+## GWA-CLI V3
+
+Gateway Command Line Interface Version 3 (GWA-CLI V3.0.1) is a new and enhanced tool for configuring your API Gateway settings. Key improvements include:
```
Comment (rustyjux) [inline documentation/reference/releases/2024-sept-3.0.0.md]: The supposed full name for the tool is 'gateway admin' but we never use that, only `gwa` - which is 'gwah' or 'gee-double-you-eh'
```suggestion
The `gwa` command line tool has been updated to support the changes to the API Services Portal, along with additional improvements to support Gateway Service configuration.
```

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+  * Users without any Gateways are directed to <https://api.gov.bc.ca/manager/gateways/get-started>.
+  * This page provides a basic introduction and instructions for the new Quick Start tutorial for API providers.
+  * The new Quick Start tutorial offers a simpler option to begin with before progressing to the [Protect an API with Client Credential Flow](https://developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/tutorials/protect-client-cred/) tutorial (previously called Quick Start).
+  * The revised Quick Start tutorial utilizes a new `quick-start` configuration template.
+* Additional Changes:
+  * Added breadcrumbs on Gateway management pages.
+  * **Gateways** is no longer visible in the navigation bar for non-IDIR users (BCeID, BC Services Card, GitHub).
+  * The **Namespace Access** page is renamed to **Administration Access**.
+
+## GWA-CLI V3
```
Comment (rustyjux) [inline documentation/reference/releases/2024-sept-3.0.0.md:48]: As per https://dev.developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/contribute/style-guide/#use-code-style-for-gwa-command-line-tool. And then I dropped to `v` to be in line with the format for API Services Portal. Changing to v on both would also be reasonable.
```suggestion
## `gwa` CLI 3.0.1
```

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+* Gateway selection user interface has changed.
+  * A new **My Gateways** page has been added, featuring a searchable list of Gateways.
+  * The Gateway dropdown selector has been relocated and updated. It now appears only on relevant API provider pages.
+* New starting page for new users after API provider login.
+  * Users without any Gateways are directed to <https://api.gov.bc.ca/manager/gateways/get-started>.
+  * This page provides a basic introduction and instructions for the new Quick Start tutorial for API providers.
+  * The new Quick Start tutorial offers a simpler option to begin with before progressing to the [Protect an API with Client Credential Flow](https://developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/tutorials/protect-client-cred/) tutorial (previously called Quick Start).
+  * The revised Quick Start tutorial utilizes a new `quick-start` configuration template.
+* Additional Changes:
+  * Added breadcrumbs on Gateway management pages.
```
Comment (rustyjux) [inline documentation/reference/releases/2024-sept-3.0.0.md:43]:
```suggestion
```
Not necessary

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+  * **Gateways** is no longer visible in the navigation bar for non-IDIR users (BCeID, BC Services Card, GitHub).
+  * The **Namespace Access** page is renamed to **Administration Access**.
+
```
Comment (rustyjux) [inline documentation/reference/releases/2024-sept-3.0.0.md:47]:
```suggestion

!!! note "New Gateway creation"
    Whereas previously it was possible to create a new Namespace via the `gwa` CLI or API Services Portal web interface, it is now _only_ possible to create Gateways using `gwa`.

```

**Review by rustyjux — CHANGES_REQUESTED** [review]
A few suggested changes to provide more or more accurate context in the intro sentence, and reducing some unnecessary details for end users. Hopefully my comments line up still - Tim pushed some line wrapping changes while I was mid-review.

When it comes to writing communication to send out to users, I think we will want to also include a 'What do I need to do' section which reinforces that users don't need to do anything and that all existing Namespaces are preserved and now called Gateways.

File: documentation/reference/releases/2024-sept-3.0.0.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+* Additional Changes:
+  * **Gateways** is no longer visible in the navigation bar for non-IDIR users
+    (BCeID, BC Services Card, GitHub).
+  * The **Namespace Access** page is renamed to **Administration Access**.
+
+!!! Note:- "New gateway creation"
+
+* Whereas previously it was possible to create a new Namespace via the `gwa`
+CLI or API Services Portal web interface, it is now _only_ possible to create
+Gageways using `gwa`.
```
Comment (rustyjux) [inline documentation/reference/releases/2024-sept-3.0.0.md:54]: This formatting will not work and also breaks with the style guide. Same with the changes you made to the `!!! note` line. Please use the changes I provided before as is and review the style guide for usage on callouts https://dev.developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/contribute/style-guide/#callouts
Previewing locally will reveal these issues or if syntax is correct.

You also have a typo and wrote 'Gageways'

**Review by rustyjux — APPROVED** [review]

**Pull request merged**